alpha/declcfg/write.go

```diff
@@ -202,8 +202,10 @@ func (writer *MermaidWriter) WriteChannels(cfg DeclarativeConfig, out io.Writer)
 		out.Write([]byte("  end\n"))
 	}
 
-	if deprecatedPackage != "" {
-		out.Write([]byte(fmt.Sprintf("style %s fill:#989695\n", deprecatedPackage)))
+	if len(deprecatedPackages) > 0 {
```

Comment:
question... is there any reason that we need to check the size of the slice?  If we've encountered no matching deprecations the loop would become a NOOP, right?

(No need to take out, because we're doing the same for 211, but I wanted to grab an opportunity to ask.)